### PR TITLE
Set suiteName on FLUID_TEST_MULTIREPORT

### PR DESCRIPTION
**Pull request by Abe27342**
## Description

Sets suiteName on `FLUID_TEST_MULTIREPORT` similarly to what we did on `FLUID_TEST_REPORT`. This restores some of the niceties in e2e test names we had before switching flags in #15983

### CODE REVIEW

**Comment by Abe27342** [comment]
note: I'd love to see feature parity between FLUID_TEST_REPORT and FLUID_TEST_MULTIREPORT and then usages of the previous flag cleaned up. Not going to attempt that here.

I'm also not sure this goal of #15983 was actually realized (I haven't been seeing failures in console):

> This PR attempts to improve that by adding test failure logging to the console output during the run.

**Comment by curtisman** [comment]
Should we just merge the two and do MULTIREPORT all the time.   The reason for the separate REPORT mode exist is because it doesn't output to console, so it is not suitable for local usage.

**Comment by curtisman** [comment]
> note: I'd love to see feature parity between FLUID_TEST_REPORT and FLUID_TEST_MULTIREPORT and then usages of the previous flag cleaned up. Not going to attempt that here.
> 
> I'm also not sure this goal of #15983 was actually realized (I haven't been seeing failures in console):
> 
> > This PR attempts to improve that by adding test failure logging to the console output during the run.

I think this is refer to the extra step in the pipeline to parse the JSON file and output them added in PR#14702.

Although, why not add "console" reporter as well, I wonder.

**Review by curtisman — APPROVED** [review]

**Comment by Josmithr** [comment]
> > note: I'd love to see feature parity between FLUID_TEST_REPORT and FLUID_TEST_MULTIREPORT and then usages of the previous flag cleaned up. Not going to attempt that here.
> > I'm also not sure this goal of #15983 was actually realized (I haven't been seeing failures in console):
> > > This PR attempts to improve that by adding test failure logging to the console output during the run.
> 
> I think this is refer to the extra step in the pipeline to parse the JSON file and output them added in PR#14702.
> 
> Although, why not add "console" reporter as well, I wonder.

Yeah, my PR was a follow-up to #14702, which opted to use the JSON reporter. I'm not sure what the motivation was for using that rather than a console reporter. @zhenmichael do you have context on this?

File: packages/test/mocha-test-setup/mocharc-common.js [hunk 1]
```diff
@@ -77,17 +77,23 @@ function getFluidTestMochaConfig(packageDir, additionalRequiredModules, testRepo
...
 		// See https://www.npmjs.com/package/mocha-multi-reporters#cmroutput-option
 		const outputFilePrefix = testReportPrefix !== undefined ? `${testReportPrefix}-` : "";
 		console.log(
 			`Writing test results relative to package to nyc/${outputFilePrefix}junit-report.xml and nyc/${outputFilePrefix}junit-report.json`,
 		);
+		const suiteName =
+			testReportPrefix !== undefined
+				? `${packageJson.name} - ${testReportPrefix}`
+				: packageJson.name;
+		console.log("suite name", suiteName);
```
Comment (Josmithr) [inline packages/test/mocha-test-setup/mocharc-common.js:91]: Should we remove this?

**Review by Josmithr — APPROVED** [review]

**Pull request merged**